src/RTG/ChatFilter/Loader.php

```diff
@@ -33,64 +33,41 @@
 use pocketmine\event\player\PlayerChatEvent;
 
 class Loader extends PluginBase implements Listener {
-
-    public $words;
+    public $cfg;
```

Comment:
You actually have a field $this->cfg, but you declared it dynamically. Every field that you know to exist should actually be defined, although there is no major impact if you don't.